core/src/Library.js

```diff
@@ -69,10 +69,20 @@ const Router = typeof document !== 'undefined'
   : StaticRouter
 
 export class Library extends React.Component {
+  static propTypes = {
+    baseUrl: PropTypes.string
+  }
+
+  static defaultProps = {
+    baseUrl: '/'
+  }
+
   render () {
+    const { baseUrl, ...props } = this.props
+
     return (
-      <Router context={{}}>
-        <LibraryApp {...this.props} />
+      <Router basename={baseUrl} context={{}}>
```

Comment:
Can we use the same prop name, e.g. `basename` – x0 and kit cli will probably use the `basename` option as well

Reply:
Keeping it as a passthrough prop also means that the Router components propTypes will make the one in Library unnecessary